### ISSUE TITLE
Will onnx_ptq support smoothquant in future?

### ISSUE THREAD

**Issue by tp-nan** [NVIDIA/TensorRT-Model-Optimizer#58]
Hi, guys, Im wondering would smoothquant be supported in the future for int8 *onnx* quant?
Mainly for VIT-like model and LLM.

**Comment by i-riyad** [comment]
Yes we plan to include it on modelopt 0.19 (sometime in Oct 2024).
